scanpipe/pipes/purldb.py

```diff
@@ -49,7 +49,7 @@
 if PURLDB_API_KEY:
     session.headers.update({"Authorization": f"Token {PURLDB_API_KEY}"})
 
-DEFAULT_TIMEOUT = 30
+DEFAULT_TIMEOUT = 120
```

Comment:
I'm a bit concerned with such a high timeout value. Having to deal with time-consuming requests may be a source of issues in the future. It would be better to deal with smaller requests with a shorter execution time.

Reply:
I've lowered this to 60 now